.gitignore

```diff
@@ -10,3 +10,4 @@ _scratch/
 /spec/gen_spec/spec*.db
 tags
 /tests/debug.py
+venv
```

Comment:
A git-ignore particular to your personal development environment belongs in your own gitignore file, not the project's. You find that somewhere like `~/.config/git/ignore` or perhaps `~/.gitconfig`.

Reply:
Thank you for that, will check it and update.